annif/annif.py

```diff
@@ -54,9 +55,24 @@ def listprojects():
 
     REST equivalent: GET /projects/
     """
-    projs = es.search(index=INDEX_NAME)['hits']
-    print(projs)
-    pass
+
+    doc = {
+            'size': 1000,
+            'query': {
+                'match_all': {}
+                }
+            }
+    results = es.search(index=annif.config['INDEX_NAME'], doc_type='project',
+            body=doc)
+    projects = [x['_source'] for x in results['hits']['hits']]
+    for p in projects:
+        print(p)
+
+    print(projects)
+    report = cat.indices()  # This queries different indices and returns
```

Comment:
Is this really necessary? The result seems to be just thrown away.

Reply:
Not really, same thing applies here. I'll do another commit to remove these before merging.